src/locale/locales/fr/messages.po

```diff
@@ -1194,20 +1132,20 @@ msgstr "Tapotez deux fois pour vous connecter"
 
 #: src/view/screens/Settings/index.tsx:755
 msgid "Download Bluesky account data (repository)"
-msgstr ""
+msgstr "Télécharger les données du compte Bluesky (dépôt)"
 
 #: src/view/screens/Settings/ExportCarDialog.tsx:59
 #: src/view/screens/Settings/ExportCarDialog.tsx:63
 msgid "Download CAR file"
-msgstr ""
+msgstr "Télécharger le fichier CAR"
 
 #: src/view/com/composer/text-input/TextInput.web.tsx:249
 msgid "Drop to add images"
-msgstr ""
+msgstr "Déposer pour ajouter des images"
 
 #: src/screens/Onboarding/StepModeration/AdultContentEnabledPref.tsx:111
 msgid "Due to Apple policies, adult content can only be enabled on the web after completing sign up."
-msgstr ""
+msgstr "En raison des politiques d’Apple, le contenu pour adultes ne peut être activé que via le Web après l’enregistrement est terminé."
```

Comment:
`inscription` is used more often than `enregistrement` in the context of apps and web services that you sign in/up. Also, I rewrote the sentences so it was less… grammatically unsound? (not judging, I do this kind of grammatical weirdnesses in English _all the time_)

```suggestion
msgstr "En raison des politiques d’Apple, le contenu pour adultes ne peut être activé que via le Web une fois l’inscription terminée."
```